domains/vasu.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Vasu's personal developer website",
+  "repo": "https://github.com/vasu-xd/VasuXD/",
+  "owner": {
+    "username": "Vasu-xD",
+    "email": "mr.v4su@gmail.com"
+    "telegram": "VasuXD",
+    "github": "VasuXD"
+  },
+  "record": {
+    "CNAME": "vasu-xd.github.io"
```

Comment:
I can't seem to reach this site.

Reply:
now okay! 

👉🏻 https://vasu-xd.github.io/Vasu-xD/